### PR TITLE
Yyx/123 reset tabs

### PR DESCRIPTION
**Pull request by yixiangyin**
# Pull Request Details

## What issue does this PR address

- reset to dataset tab

- Link to associated issue: #123 

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [ ] Screenshots included in linked issue
- [ ] Changes adhere to the style and coding guideline
- [ ] No confidential information
- [ ] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [ ] Pre-exisiting lint errors noted: [HERE]
- [x] Tested on at least one device
  - [ ] Android Phone
  - [ ] Android Emulator
  - [ ] Chrome on Android
  - [ ] Chrome
  - [ ] iOS
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [ ] Added a reviewer

## Finalising

Once PR discussion is complete and reviewer has approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Comment by yixiangyin** [comment]
home_tabs.dart is not used. Should we remove it in this pr?

**Review by gjwgit — CHANGES_REQUESTED** [review]
This pull request is "advertised" as fixing the reset tabs (essentially reset to the DATASET tab on a REST tap).

That works just fine and thanks for that.

However, this PR also includes some work in progress for #121. Could that work be split into a different branch please for separate discussion. Thanks. It looks great so far and I'll comment on some ideas in the issue #121.

Then I can merge this PR for #123.

File: lib/panels/cluster/config.dart
Comment (gjwgit) [inline lib/panels/cluster/config.dart:1]: Thanks for fixing these.

File: lib/panels/tree/display.dart
Comment (gjwgit) [inline lib/panels/tree/display.dart:1]: Perhaps this into a different branch specific for panel navigation.

File: lib/panels/tree/display.dart [hunk 1]
```diff
@@ -45,25 +45,108 @@ class TreeDisplay extends ConsumerStatefulWidget {
...
+          ),
+          onPressed: _currentPage < numPages - 1
+              ? _goToNextPage
+              : null,
+        ),
+      ],
+    );
   }
-}
+}
```
Comment (gjwgit) [inline lib/panels/tree/display.dart:152]: You can set you IDE to always include a final new line in your files.

**Comment by gjwgit** [comment]
I'll approve and merge this one with the extra "feature" but do aim to keep them as separate branches and PRs. Thanks

**Comment by yixiangyin** [comment]
> This pull request is "advertised" as fixing the reset tabs (essentially reset to the DATASET tab on a REST tap).
> 
> That works just fine and thanks for that.
> 
> However, this PR also includes some work in progress for #121. Could that work be split into a different branch please for separate discussion. Thanks. It looks great so far and I'll comment on some ideas in the issue #121.
> 
> Then I can merge this PR for #123.

Apologies I just realised that. I will create one for 121.

**Pull request merged**